subscriber/podaac_access.py

```diff
@@ -550,3 +551,18 @@ def create_citation_file(short_name, provider, data_path, token=None, verbose=Fa
 
     with open(data_path + "/" + short_name + ".citation.txt", "w") as text_file:
         text_file.write(citation)
+
+def get_latest_release():
+    github_url = "https://api.github.com/repos/podaac/data-subscriber/releases/latest"
```

Comment:
Being a little nit-picky but `releases/latest` returns the latest by created date not necessarily the newest version (for example if we were to release a patch to a previous version). We'd probably have to list all releases and sort to get the highest version number to find the latter.

Reply:
i'm all for this, my concern is that there is no guarantee that https://api.github.com/repos/podaac/data-subscriber/releases is in order (most recent to last) even though it appears that way. i just worry about needing to _page_ through the releases at some point. 

https://api.github.com/repos/podaac/data-subscriber/releases

Reply:
it appears to be well behaved:

```
curl https://api.github.com/repos/podaac/data-subscriber/releases | jq .[].tag_name
"1.12.0"
"1.11.0"
"1.10.2"
"1.10.1"
"1.10.0"
"1.9.1-r2"
"1.9.0"
"1.8.0"
"1.7.2"
"1.7.1"
"1.7.0"
```

Reply:
Throw a sort in there for good measure:
```
version_list = ["1.12.0",
"1.11.0",
"1.10.2",
"1.10.1",
"1.10.0",
"1.9.1-r2",
"1.9.0",
"1.8.0",
"1.7.2",
"1.7.1",
"1.7.0"]
random.shuffle(version_list)

list(sorted([version.parse(v) for v in version_list]))[-1]

>>> <Version('1.12.0')>
```